### PR TITLE
Feat/pieces 8.1.0

### PR DESCRIPTION
**Pull request by caleb-at-pieces**
bumps the sdk to the latest in main

### CODE REVIEW

File: core/runtime.ts
Comment (mark-at-pieces) [inline core/runtime.ts:1]: lol i see u sneaking this in here

**Review by mark-at-pieces — APPROVED** [review]
LGTM

File: core/runtime.ts
Comment (caleb-at-pieces) [reply core/runtime.ts:1]: yea the code itself is fine it just throws a fit because of browser / server stuff

Comment (mark-at-pieces) [reply core/runtime.ts:1]: yeah ikik u all gucci

**Pull request merged**